### DISCUSSION TITLE
Error while executing the example "TB2J with Wannier90"

### DISCUSSION THREAD

**Discussion by rajeshatiitm** [mailhexu/TB2J#20] [General]
Dear XU,

I was executing the example code (SrMnO3) as explained here https://tb2j.readthedocs.io/en/latest/src/wannier.html

I am getting an error which I couldn't figure out. 
_---------------------------------------------------------------

  Calculation of overlap and call to wannier90 library 
  to obtain maximally localized wannier functions 

  - abinito_w90_up.win and abinito_w90_down.win
are mandatory secondary input
  - abinito_w90_up.wout and abinito_w90_down.wout
 are the output for the library
  - abinito_w90random_up.amn and abinito_w90random_down.amn
 contain random projections
  - abinito_w90_up.amn and abinito_w90_down.amn
 contain projections
  - abinito_w90_up.mmn and abinito_w90_down.mmn
 contain the overlap
  - abinito_w90_up.eig and abinito_w90_down.eig
 contain the eigenvalues

---------------------------------------------------------------

   mlwfovlp:  you should give k-point in the full brillouin zone 
   with explicit k-points (or kptopt=3) and istwfk 1

   mlwfovlp:  you should give k-point in the full brillouin zone 
   with explicit k-points (or kptopt=3) and istwfk 1

** mlwfovlp_setup:  call wannier90 library subroutine wannier_setup
 Unrecognised keyword(s) in input file, see also output file
 Error on node 0: examine the output/error files for details
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them._

I am using ABINIT v8.10.3 and the Wannier v2.0.1.1 which was installed from abinit fallbacks. 

I have used the exact same input file in the SrMnO3 folder, did not change any thing. kptopt has been set to 3 and istwfk set to 64*1

Any help please?

**Comment by mailhexu** [comment]
Hi, 
This is actually in the documented:
 https://tb2j.readthedocs.io/en/latest/src/wannier.html

In the Wannier90 ".win" file:
# write the postitions of WF
write_xyz = true
# write the WF Hamiltonian **(Note for W90 version<2.1, it is hr_plot)**
write_hr = true

There are some changes of the keywords in Wannier90 version 2.1. And the example is for the new version by default. 

You can either update Abinit/Wannier90 to new version (which is highly recommended as there are a lot of new features and bug fixes in Abinit and Wannier during the years), or change  write_hr to hr_plot. 

Best regards,
HeXu
